csrc/flash_attn_rocm/fmha_api.cpp

```diff
@@ -102,6 +119,22 @@ void set_params_fprop(FlashFwdParams &params,
         int temp_q_stride = get_size_in_bytes(d * h * temp_seqlen_q, data_type);
         int temp_seqlen_k = params.host_seqlens_k[i+1] - params.host_seqlens_k[i];
         int temp_k_stride = get_size_in_bytes(d * h * temp_seqlen_k, data_type);
+
+        if(!params.is_mnko_padding && d <= 32){
+            params.is_mnko_padding = ((temp_seqlen_q % 128)==0 && (temp_seqlen_k % 128)==0 ? false : true);
```

Comment:
Check the mod of total length with 128 or 256 could get is_mnko_padding directly.

Reply:
Yes, the implementation is correct.